package.json

```diff
@@ -34,10 +34,16 @@
     "@ceramicnetwork/http-client": "^0.8.8",
     "@types/jest": "^26.0.19",
     "babel-jest": "^26.6.3",
+    "ipfs-core": "0.3.0",
+    "ipfs-http-client": "48.1.1",
     "jest": "^26.6.3",
     "key-did-provider-ed25519": "^1.0.0",
+    "multiformats": "3.0.3",
     "node-config-ts": "^3.1.0",
-    "ts-jest": "^26.4.4"
+    "ts-jest": "^26.4.4",
```

Comment:
The distinction between `dependencies` and `devDependencies` is pretty fuzzy here since we don't expect anyone else to ever actually import this package.  Do you have a suggestion for which packages belong in `devDependencies` and which in `dependencies`?

Reply:
Yeah fair enough. I think it's fine without in this case